### PR TITLE
Migrate to `Text`, `Skeleton`; fix text in titles and labels

### PR DESCRIPTION
**Pull request by Cloud11PL**
## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [ ] 🛠 Workflow CI/CD changes

## Related Issues or Documents

- closes #

## Have you written tests?
- [ ] Yes!
- [ ] No... here is why: _Writing tests are mandatory, please replace this text with why test are not included in this PR_

### CODE REVIEW

**Review by lkostrowski — DISMISSED** [review]

**Review by poulch — APPROVED** [review]
LGTM, but I've found that there are many Text component with custom styles passed as className. I'd suggest to get rid of all styles passed as className from makeStyle.

File: src/components/EditableTableCell/EditableTableCell.tsx [hunk 1]
```diff
@@ -70,9 +71,9 @@ export const EditableTableCell: React.FC<EditableTableCellProps> = props => {
   return (
     <TableCell className={clsx(classes.container, className)}>
       {opened && <div className={classes.overlay} onClick={disable} />}
-      <Typography variant="caption" onClick={enable} className={classes.text}>
+      <Text size={2} fontWeight="light" onClick={enable} className={classes.text}>
```
Comment (poulch) [inline src/components/EditableTableCell/EditableTableCell.tsx:74]: question: Didn't you lost color here?

**Review by poulch — APPROVED** [review]

File: src/components/EditableTableCell/EditableTableCell.tsx (hunk 1 above)
Comment (Cloud11PL) [reply src/components/EditableTableCell/EditableTableCell.tsx:74]: I (surprisingly) didn't. Current implementation of Material theme makes caption have the same colour as the rest

**Review by krzysztofzuraw — APPROVED** [review]

**Comment by Cloud11PL** [comment]
E2E will be fixed here https://github.com/saleor/saleor-dashboard/pull/5052

**Pull request merged**